### ISSUE TITLE
Trying to reshare a file results in 403, but file is still re-shared

### ISSUE THREAD

**Issue by mario** [nextcloud/spreed#1449]
{"ocs":{"meta":{"status":"failure","statuscode":403,"message":"Path is already shared with this room"},"data":[]}}

**Comment by nickvergessen** [comment]
Yes, thats expected behaviour, iirc.

**Comment by nickvergessen** [comment]
Yeah well its the share API and there the behaviour is expected. We just also listen to the error hook and still add the message, but in fact the item can not be shared again

**Comment by mario** [comment]
Feel free to close the issue then. :)
